Kernel/FileSystem/ProcFS.cpp

```diff
@@ -604,7 +604,7 @@ struct SysVariableData final : public ProcFSInodeCustomData {
         Boolean,
         String,
     };
-    Type type { Invalid };
+    Type type{ Invalid };
```

Comment:
Hmm, it looks like you're running an older clang-format. It's not supposed to do this.